distribution.md

```diff
@@ -228,3 +230,89 @@ The Run Image SHOULD contain the following Labels on the image configuration:
 Where,
 
 `<Target ID>` is an identifier specified on the runtime image that MAY be used to apply target-specific logic.
+
+### Builder
+
+The following specifies the artifact format for a buildpacks builder.
+
+A builder is an OCI Image that provides a distributable build environment.
+
+A platform supporting builders SHOULD allow users to configure the build environment for a provided builder.
+
+#### General Requirements
+
+The builder image MUST contain an implementation of the [lifecycle](#lifecycle), and a [build-time](#build-image) environment, and MAY contain [buildpacks](#buildpackage). Platforms SHOULD use builders to ease the build process.
+
+#### Filesystem
+
+A builder MUST have the following directories/files:
+
+* `/cnb/order.toml` &rarr; As defined in the [platform specification](https://github.com/buildpacks/spec/blob/main/platform.md#ordertoml-toml)
+
+#### Environment Variables
+
+A builder MUST be an extension of a Build Image, and MUST retain all the specified environment variables set on the original build image, as specified in the Build Image specifications.
+
+The following environment variables MAY be set on the builder (through the image config's `Env` field):
+
+| Env Variable             | Description                                                                       | Default
+| --------                 | --------                                                                          |-----------
+| `CNB_APP_DIR`            | Application directory of the build environment                                    | `/workspace`
+| `CNB_LAYERS_DIR`         | The directory to create and store `layers` in the build environment               | `/layers`
+| `CNB_PLATFORM_DIR`       | The directory to create and store platform focused files in the build environment | `/platform`
+| `SERVICE_BINDING_ROOT`   | The directory where services are bound                                            | -
```

Comment:
I don't think including defaults make sense here. The defaults are a piece of lifecycle behavior, specified in the platform API and don't belong in this specification

Reply:
Resolved in 35e5cce6acb83cfa018f61dc0bc90019375b8702